system-parachains/asset-hub-paseo/src/lib.rs

```diff
@@ -896,6 +920,7 @@ construct_runtime!(
 		XcmpQueue: cumulus_pallet_xcmp_queue = 30,
 		PolkadotXcm: pallet_xcm = 31,
 		CumulusXcm: cumulus_pallet_xcm = 32,
+		// DmpQueue: cumulus_pallet_dmp_queue = 33, removed
```

Comment:
Probably ok to leave out :)

Reply:
yeap, i added it on purpose to not have the diff when we bring the polkadot runtime in future releases.